lib/sanitizerPolicies.js

```diff
@@ -0,0 +1,33 @@
+function addTabindexToIdPolicy(node) {
```

Comment:
Please sync with phoenix-ui version of this file.

Reply:
changed